python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -58,6 +58,8 @@ export type AIConfigCallbacks = {
   ) => Promise<{ aiconfig: AIConfig }>;
 };
 
+type RequestCallbackError = { message?: string };
```

Comment:
I see, so this is kind of like taking a generic `Error` object and casting it to an object where the only thing we care about is the message field if it exists? Not sure how much you plan to extend it, but for now would prefer to have this be more obvious in the name. Ex: `RequestCallbackErrorWithMaybeMessage`. Kind of long and tedious naming, so you don't have to do this if you don't want to

Reply:
It's a bit strange how we need to handle it -- basically, any server response that is a `HttpResponseWithAIConfig` with a code will result in a fetch request error but the caught value is not an error, it's just a raw object containing message. However -- it's still possible that an Error could be caught, or I guess some other arbitrary type. So, just typing with optional message to be safe

Reply:
Going to leave as-is but if you feel super strongly on this please lmk and I can fix forward